javascript/javascript1/week1/ez-namey.js

```diff
@@ -0,0 +1,10 @@
+//Ez Namey
+
+const firstWordsArray = ['Easy', 'Corporate', 'Awesome', 'Tech', 'Business', 'Creative', 'Data', 'Alpha', 'Delta', 'Cloud'];
+const secondWordsArray = ['Beta', 'Infinite', 'Level', 'Collective', 'Collaboration', 'Server', 'Craft', 'Brand', 'Vital', 'Blink'];
+
+const randomNumber = Math.floor(Math.random() * 10) + 0;
+const startUpName = firstWordsArray[randomNumber] + ' ' + secondWordsArray[randomNumber];
```

Comment:
These will both use the same number, because the random number is only calculated once per run. 
Just so you're aware.

Reply:
I didn't know that- thank you. If I would like to get a separate number for each of the array- what should i do?

Reply:
Declare 2 different random numbers (`randomNumber1` and `randomNumber2`).

Reply:
Thank you- I updated my file with the different numbers now :)